pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: gcloud
-version: 0.6.0+1
+version: 0.6.0+2
 author: Dart Team <misc@dartlang.org>
 description: Dart gcloud APIs
 homepage: https://github.com/dart-lang/gcloud
+
 environment:
-  sdk: '>=2.0.0-dev.54.0 <3.0.0'
+  sdk: '>=2.0.0 <3.0.0'
+
 dependencies:
   _discoveryapis_commons: ^0.1.6+1
```

Comment:
Unrelated to this PR, but I see that `package:_discoveryapis_commons` is currently at `0.1.8+1`. Do you happen to know if this package requires a specific version, or if the range can be broadened?

Reply:
This constraint includes the latest _discoveryapis_commons

Reply:
Ah, yeah; I read that constraint wrong.